connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -173,10 +174,16 @@ class SparkConnectPlanner(plan: proto.Relation, session: SparkSession) {
     }
   }
 
-  private def transformFetch(limit: proto.Fetch): LogicalPlan = {
+  private def transformLimit(limit: proto.Limit): LogicalPlan = {
     logical.Limit(
-      child = transformRelation(limit.getInput),
-      limitExpr = expressions.Literal(limit.getLimit, IntegerType))
+      limitExpr = expressions.Literal(limit.getLimit, IntegerType),
+      transformRelation(limit.getInput))
+  }
+
+  private def transformOffset(offset: proto.Offset): LogicalPlan = {
+    Offset(
```

Comment:
please use `logical.Offset` to make sure where it comes from similar to the `proto.Offset`

Reply:
+1